src/gui/tray/Window.qml

```diff
@@ -21,9 +21,6 @@ Window {
 
     readonly property int maxMenuHeight: Style.trayWindowHeight - Style.trayWindowHeaderHeight - 2 * Style.trayWindowBorderWidth
 
-    Accessible.role: Accessible.Application
-    Accessible.name: qsTr("Nextcloud desktop main dialog")

```

Comment:
@camilasan Is this really not needed anymore?

Reply:
This was throwing a warning: Accessible must be attached to an Item. And look at lines 90 and 91: the exact same Accessible was already defined there.